README.md

```diff
@@ -41,6 +41,12 @@ cd opentelemetry-demo/
 
 #### Run Docker Compose
 
+- Build an image that downloads required files
```

Comment:
How will this integrate with the CI / CD? Do I need to add something to a GitHub workflow?